docs/docs.md

```diff
@@ -0,0 +1,131 @@
+# Documentation
+
+## Supported Uptime Checkers
```

Comment:
@ahmadiq lets keep this part in the README; just list of supported uptime checkers

Reply:
Done